example/pythonlib/web/2-todo-flask/build.mill

```diff
@@ -0,0 +1,62 @@
+// This is a `Flask`-based `TodoMVC` application managed and built using `Mill`.
```

Comment:
This example and the other django equivalent below should link to https://todomvc.com/.

Reply:
done